packages/flutter_tools/lib/src/run_cold.dart

```diff
@@ -158,7 +158,7 @@ class ColdRunner extends ResidentRunner {
   void printHelp({ @required bool details }) {
     bool haveDetails = false;
     if (_result.hasObservatory)
-      printStatus('The Observatory debugger and profiler is available at: ${_result.observatoryUri}');
+      printStatus('The Observatory debugger and profiler is available at: ${_result.observatoryUri}.');
```

Comment:
the lack of period here is intentional, we don't want the url to get a trailing period.
The grammar is fine, the `:` ends the sentence.